tools/importer/rules/marquee.js

```diff
@@ -0,0 +1,104 @@
+/* global WebImporter */
+export default function createMarqueeBlocks(main, document) {
+  const marquees = document.querySelectorAll('#marquee');
+
+  // fast return
+  if (!marquees) {
+    return;
+  }
+
+  marquees.forEach((marquee) => {
+    // Create table for marquee
+    const cells = [['Marquee']];
+
+    // Check if marquee has background video
+    if (marquee.querySelector('.has-video.dexter-Background')) {
+      const backgroundVideoURL = marquee.querySelector('.has-video.dexter-Background').querySelector('video').querySelector('source').getAttribute('src');
+      const bgvideoa = document.createElement('a');
+      bgvideoa.innerHTML = backgroundVideoURL;
+      bgvideoa.setAttribute('href', backgroundVideoURL);
+      cells.push([bgvideoa]);
+    }
+
+    // Check if marquee has icon
+    const images = marquee.querySelectorAll('img');
+    let iconImageUrl = null;
+    let iconImage = null;
+    images.forEach((image) => {
+      if (iconImage) {
+        return;
+      }
+      if (image.getAttribute('alt').indexOf('icon') > -1) {
+        if (image.getAttribute('src').indexOf('https') === -1) {
+          iconImage = `https://www.adobe.com/${image.getAttribute('src')}`;
+        } else {
+          iconImage = image.getAttribute('src');
+        }
+        iconImageUrl = document.createElement('a');
+        iconImageUrl.innerHTML = iconImage;
+        iconImageUrl.setAttribute('href', iconImage);
+      }
+    });
```

Comment:
```suggestion
      const alt = image.getAttribute('alt');
      const src = image.getAttribute('src');
      if (alt && alt.indexOf('icon') > -1) {
        if (src.indexOf('https') === -1) {
          href = `https://www.adobe.com/${src}`;
        } else {
          href = src;
        }
        iconImageLink = document.createElement('a');
        iconImageLink.innerHTML = href;
        iconImageLink.setAttribute('href', href);
      }
    });
```

Probably a bit more to change above.

Reply:
Done